### PR TITLE
add XYLabels

### PR DESCRIPTION
**Pull request by ctessum**

### CODE REVIEW

File: plotter/labels.go [hunk 1]
```diff
@@ -115,3 +112,20 @@ func (l *Labels) GlyphBoxes(p *plot.Plot) []plot.GlyphBox {
 	}
 	return bs
 }
+
+// XYLabeller combines the XYer and Labeller types
```
Comment (eaburns) [inline plotter/labels.go]: Add a period at the end.

File: plotter/labels.go [hunk 2]
```diff
@@ -115,3 +112,20 @@ func (l *Labels) GlyphBoxes(p *plot.Plot) []plot.GlyphBox {
...
+type XYLabeller interface {
+	XYer
+	Labeller
+}
+
+// XYLabels holds XY data with labels.
```
Comment (eaburns) [inline plotter/labels.go]: Can you add a note on how the labels match up with the XYs?
Maybe something like, "The ith label corresponds to the ith XY."?

**Comment by eaburns** [comment]
This LGTM. I don't remember why I didn't do it in the first place, but I remember consciously avoiding it. I think that I was just trying to cut down on the number of interfaces. But this seems fine to me.

**Comment by ctessum** [comment]
Looks like I mis-merged again. Sorry about that.

**Pull request closed without merging**